pkg/defaults/policies/files/severity_high_privileged.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "b1df1abb-e5a5-4ff7-98fe-1d28a22b55d8",
-  "name": "Privileged Containers with Important and Critical Fixable CVEs",
-  "description": "Alert on containers running in privileged mode with important or critical fixable vulnerabilities",
+  "name": "Privileged containers with fixable CVEs and Severity at least Important",
+  "description": "Alert on containers running in privileged mode with fixable vulnerabilities with Severity rating Important or higher. If you prefer CVSS-based policy, consider enabling 'Privileged Containers with Fixable CVEs and CVSS >= 6' instead.",
```

Comment:
alerts for policy violations are raised on deployment level. hence and descriptions  should be worded accurately